### PR TITLE
refactor the interface of Crud Precompiled Contract

### PR DESCRIPTION
**Pull request by sulenn**

### CODE REVIEW

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/crud/crud_service_test.go  1
         

Clones added
============
- precompiled/crud/crud_service.go  3
- precompiled/permission/permission_service.go  1
- precompiled/crud/crud_service_test.go  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5878769&bid=19240279)

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/crud/crud_service_test.go  1
         

Clones added
============
- precompiled/permission/permission_service.go  1
- precompiled/crud/crud_service.go  3
- precompiled/contractlifecycle/contract_life_cycle_test.go  4
- precompiled/chaingovernance/chain_governance_test.go  5
- precompiled/crud/crud_service_test.go  3
         

Clones removed
==============
+ precompiled/permission/permission_service_test.go  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5878769&bid=19240279)

File: precompiled/utils.go [hunk 1]
```diff
@@ -7,14 +7,14 @@ import (
...
-	"github.com/FISCO-BCOS/go-sdk/core/types"
-
 	"github.com/FISCO-BCOS/go-sdk/abi"
+	"github.com/FISCO-BCOS/go-sdk/core/types"
 )
 
 const (
 	// common precompiled contract error code
-	noAuthorized int64 = -50000
+	noAuthorized          int64 = -50000
```
Comment (bxq2011hust) [inline precompiled/utils.go:16]: permissionDenied

File: precompiled/utils.go [hunk 2]
```diff
@@ -7,14 +7,14 @@ import (
...
+	tableNameAlreadyExist int64 = -50001
```
Comment (bxq2011hust) [inline precompiled/utils.go:17]:
```suggestion
	tableAlreadyExist int64 = -50001
```

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/crud/crud_service_test.go  1
         

Clones added
============
- precompiled/crud/crud_service_test.go  3
- precompiled/chaingovernance/chain_governance_test.go  5
- precompiled/permission/permission_service.go  1
- precompiled/contractlifecycle/contract_life_cycle_test.go  4
- precompiled/crud/crud_service.go  3
         

Clones removed
==============
+ precompiled/permission/permission_service_test.go  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5878769&bid=19240279)

**Pull request merged**